### PR TITLE
Minor edits + this weird json thing I don't understand

### PR DESCRIPTION
**Pull request by Mbjoerkh**

### CODE REVIEW

**Comment by Vandivier** [comment]
package-lock.json isn't needed in the repo

File: js/earhart-fellows/EarhartFellowsMergededited.txt [hunk 1]
```diff
@@ -25423,8 +25423,8 @@ Bennett, Sponsor, Ph.D., 1956
 WOOD, Carolyn J.
 Graduate Fellowship(s)
 1973-1974
-Washington University, Education, Earhart
-Foundation, Sponsor, Ph.D.
+Washington University, Education,
+EarhartFoundation, Sponsor, Ph.D.
```
Comment (Vandivier) [inline js/earhart-fellows/EarhartFellowsMergededited.txt]: did you mean to collapse the space here?

File: js/earhart-fellows/output.csv [hunk 2]
```diff
@@ -1800,7 +1800,7 @@
 "NOWLIN, Jack Wade","1998-1999","","Princeton University","","Government/Politics","","","Robert P. George","Ph.D.","1999","School of Law, Robert C. Khayat Law Center,University of Mississippi, P.O. Box 1848,University, MS 38677,20 Private Road 3151, Apt. 1, Oxford, MS,38655","jnowlin@olemiss.edu","true","false"
 "NOWORYTA, Timothy J.","1972-1973,1973-1974","","University of Michigan","","History","","","Stephen J.,Tonsor","","","1959 W. Summerdale Avenue, Chicago, IL,60640-1040","tnoworyta@pdiglobal.com","true","false"
 "NOYES, Chad E.","1999-2000","","Harvard University","","Government/Politics","","","Harvey C. Mansfield Jr.","Ph.D.","2004","250 Main Street, Unit 936, Ben Lomond, CA,95005-2039","cenparis98@hotmail.com","true","false"
-"NOYES, Neal E.","1974-1975,1975-1976 �","","Temple University","","Government/Politics","","","Daniel,J. Elazar","","","810 Gordon School Place, N., North,Chesterfield, VA 23236-2566,NULL, James A.","","","false"
+"NOYES, Neal E.","1974-1975,1975-1976 °","","Temple University","","Government/Politics","","","Daniel,J. Elazar","","","810 Gordon School Place, N., North,Chesterfield, VA 23236-2566,NULL, James A.","","","false"
```
Comment (Vandivier) [inline js/earhart-fellows/output.csv]: still looks like a wrong character

**Comment by Mbjoerkh** [comment]
Actually, no but it doesn’t matter.

**Comment by Mbjoerkh** [comment]
Agree with the character - I feel like some changes show up that I didn’t
make. Can this be related to using notepad++?

File: js/earhart-fellows/EarhartFellowsMergededited.txt (hunk 1 above)
Comment (Mbjoerkh) [reply js/earhart-fellows/EarhartFellowsMergededited.txt]: Nope. Fixing those now

**Comment by Mbjoerkh** [comment]
Mostly minor, then the UC, UNC, UT school system "fix"

**Comment by Mbjoerkh** [comment]
Oh, and I changed the sourcecode from Earhart to EarhartEdited

**Comment by Mbjoerkh** [comment]
If I understand this correctly I've now removed the packagelock but kept the other changes. Couldn't figure out how to remove the the collapsed "EarhartFellows" but it really doesn't matter

**Comment by Vandivier** [comment]
as discussed offlline, merging and fixing undesired changes from my side for simplicity.

Next time, please create a branch on your repo which isolates the valid pr changes. For example, your package files shouldn't go here.

**Pull request merged**